server/routes/auth.js

```diff
@@ -29,7 +29,7 @@ module.exports = [
 
       request.cookieAuth.set({ user, roles, scope })
 
-      return h.redirect(credentials.query?.redirectTo || '/account')
+      return h.redirect(credentials.query?.redirectTo || '/')
```

Comment:
There are additional instructions on the home page now, rather than sending people straight to the account page, the home page will be viewed. There are buttons on the page to get people to where they need to do afterwards.